pycon/finaid/views.py

```diff
@@ -359,6 +366,210 @@ def finaid_status(request):
 
 
 @login_required
+def finaid_accept(request):
```

Comment:
There seems to be a fair bit of repetition in these views. Would be worthwhile to distill a class-based base view?